.release_templates/pytorch_release_images.yml

```diff
@@ -0,0 +1,34 @@
+---
+release_images:
+  1:
+    framework: "pytorch"
+    version: "1.6.0"
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py36"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu101"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+                            # to images being published.
+      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+                            # has already been published. Re-released image will have minor version incremented by 1.
+    inference:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py36"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu101"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  2:
+    framework: "pytorch"
+    version: "1.6.0"
```

Comment:
nit: pytorch 1.6 isn't released yet, so it would make more sense to choose 1.5.0 (i.e. if we needed to use this today, we would have to change to a 1.5 version